src/app_errors/mod.rs

```diff
@@ -2,13 +2,16 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, you can obtain one at https://mozilla.org/MPL/2.0/.
 
-use std::collections::HashMap;
+use std::{collections::HashMap, result};
 
+use failure::Error;
 use rocket_contrib::Json;
 
 #[cfg(test)]
 mod test;
 
+pub type Result<T> = result::Result<T, Error>;
```

Comment:
May be worth calling this something like `FxaResult`, then we can use `Result` unnamespaced and we also rule out any chance of future confusion from someone mistaking our result type for the standard one. (and also it's better greppability I guess)

Reply:
Or maybe `AppResult` as this module is `app_errors`. Or something else...

Reply:
This was based on megaphone https://github.com/mozilla-services/megaphone/blob/b44f47bc11f51fe6cb4f54479cb202cec64a554c/src/error.rs#L20 but yea, this does look a bit sketchy to call this `Result`.